### PR TITLE
Alt clicking inside bag wont show turf tab

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
Changes the turf view to only open for something that is a turf or is on the turf, and not inside something else like a bag. 

## Why It's Good For The Game
Prevents tabs switching all the time.

## Changelog
:cl: Skoglol
tweak: Alt clicking will no longer show turf contents for items inside bags etc.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
can you make it not toggle walkmode either?

**Comment by kriskog** [comment]
> 
> 
> can you make it not toggle walkmode either?

No, thats entirely unrelated and also fixed in #47569

**Comment by carlarctg** [comment]
> > can you make it not toggle walkmode either?
> 
> No, thats entirely unrelated and also fixed in #47569

i don't mean that, i mean alt clicking a storage item puts you on walk mode

unrelated to hotkeys

**Comment by kriskog** [comment]
Ah. No, thats not feasible. We dont know you are holding the alt button down to alt click something before you actually click, but you chould look into moving the walk button instead. Thats what custom hotkeys are for.

File: code/_onclick/click.dm [hunk 1]
```diff
@@ -343,7 +343,7 @@
 /atom/proc/AltClick(mob/user)
 	SEND_SIGNAL(src, COMSIG_CLICK_ALT, user)
 	var/turf/T = get_turf(src)
-	if(T && user.TurfAdjacent(T))
+	if(T && (isturf(loc) || isturf(src)) && user.TurfAdjacent(T))
```
Comment (ninjanomnom) [inline code/_onclick/click.dm]: >`isturf(src)`
really shows that this needs a refactor at some point

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**